test/e2e-test/spec.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+import { exec } from 'child_process';
+import { join } from 'path';
+import { existsSync, readFileSync,writeFileSync, rmSync } from 'fs';
+import * as yaml from 'js-yaml';
+
+describe('Stencil cli e2e Test - SPEC command', () => {
+  const specFilePath = join('spec.yaml');
+  const specFileContent = `stencil: 0.0.1
+
+info:
+  properties:
+    project-name: "final"
+    package-manager: "npm"
+
+tooling: [temporal]
+
+endpoints:
+`;
+
+  it('should run the Stencil CLI with no arguments and log the error output', (done) => {
+    exec('npx stencil spec', (error, stdout, stderr) => {
+      expect(stderr).toContain("error: missing required argument 'filePath'");
+      done();
+    });
+  },);
+
+  it('should run the Stencil CLI with the "spec.yaml" file and check for directory structure', (done) => {
+    writeFileSync(specFilePath, specFileContent);
+    exec(`npx stencil spec ${specFilePath}`, (error, stdout, stderr) => {

```

Comment:
Why are we running a command via npx to test this very package?? I don't think npx will target and use this repo. How is this test case even passing on your local? Or have you pushed without even running this?